cluster/src/shard_lock_manager.rs

```diff
@@ -380,6 +411,93 @@ impl ShardLock {
         Bytes::from(key)
     }
 
+    /// The shard lock is allowed to acquired in a fast way if the lock has been
+    /// created by this node just before.
+    ///
+    /// The slow way has to wait for the previous lock lease expired. And the
+    /// fast way is to reuse the lease if it is not expired and the lock's value
+    /// is the same.
+    async fn maybe_fast_acquire_lock(&self, etcd_client: &mut Client) -> Result<Option<LeaseInfo>> {
+        if !self.enable_fast_reacquire {
+            return Ok(None);
+        }
+
+        let resp = etcd_client
+            .get(self.key.clone(), None)
+            .await
+            .context(GetLockKey {
+                shard_id: self.shard_id,
+            })?;
+        if resp.kvs().len() != 1 {
```

Comment:
Maybe it should print an error log?

Reply:
For most cases, the `len()` will be 0 or 1, and the two values are correct both.

Reply:
Hoa about adding a comment to analyze this? I think it an error when reading just code here...